ports/esp32s2/boards/electroniccats_bastwifi/mpconfigboard.mk

```diff
@@ -0,0 +1,19 @@
+USB_VID = 0x239A
+USB_PID = 0x80A6
```

Comment:
This re-uses the PID assigned to the espressif_saola_1_wrover.  It should be a distinct PID, see #2966 and https://learn.adafruit.com/how-to-add-a-new-board-to-circuitpython/customizing-the-board-files#mpconfigboard-dot-mk-3033646-25.
> USB VID and PID are something you cannot borrow from another board. So don't copy and paste these values! If you don't have a VID/PID pair, open up an issue in CircuitPython repository and request one, we will assist in the process of assigning a PID for use with CircuitPython.

Reply:
Oh sorry, fixed

Reply:
Thank you!